src/tile.cpp

```diff
@@ -888,14 +890,13 @@ void Tile::addThing(int32_t, Thing* thing)
 				// remove old splash if exists
 				for (ItemVector::const_iterator it = items->getBeginTopItem(), end = items->getEndTopItem(); it != end;
 				     ++it) {
-					Item* oldSplash = *it;
+					auto oldSplash = *it;
```

Comment:
Could this be `const auto`
```suggestion
					const auto oldSplash = *it;
```